src/index.ts

```diff
@@ -38,6 +38,15 @@ export class Factory<T> {
 
   }
 
+  public buildList(count: number, item: Partial<T>) : T[]
```

Comment:
FYI: I've got another pull request which makes the item argument optional, so after I merge this PR and the other I will make the item argument optional here as well.